src/Behat/Behat/Output/Statistics/PhaseStatistics.php

```diff
@@ -111,7 +111,7 @@ public function registerHookStat(HookStat $stat)
     /**
      * Returns counters for different scenario result codes.
      *
-     * @return array<TestResult::*, int>
+     * @return array<StepResult::*, int>
```

Comment:
this change is wrong. Scenario stats are not indexed by step result

Reply:
Actually some scenario results were indexed by the StepResult::UNDEFINED constant, that's why I introduced this change. But thinking more about this, it does not make sense for this constant to be defined in `StepResult` if it is not only used by steps, so I moved it to the base `TestResult` class and now scenario stats are only indexes by `TestResult` constants